package/template-cartesian-graph/package

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(template-cartesian-graph)
+pkgdesc="A Cartesian graph template for the reMarkable tablet"
```

Comment:
I suggest keeping the description simple. See the guidance at <https://github.com/toltec-dev/toltec/blob/stable/docs/package.md#pkgdesc>.
```suggestion
pkgdesc="Cartesian graph template"
```